projects/schematics/src/modernize-app-migrated-from-6_8-to-2211_19/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+  chain,
+} from '@angular-devkit/schematics';
+
+import { updateAngularJsonForApplicationBuilder } from './csr-and-ssr/update-angular-json-for-application-builder';
+import { updateTsConfig } from './csr-and-ssr/update-ts-config';
+import { updateAngularJsonForSsr } from './ssr/update-angular-json-for-ssr';
+import { updateTsConfigApp } from './ssr/update-ts-config-app';
+import { renameAppServerModule } from './ssr/rename-app-server-module';
+import { updateMainServerTs } from './ssr/update-main-server-ts';
+import { updateServerTs } from './ssr/update-server-ts';
+import { updateAppModule } from './csr-and-ssr/update-app-module';
+import { updatePackageJsonServerScripts } from './ssr/update-package-json-server-scripts';
+import { isUsingSsr } from './is-using-ssr';
+import { removeTsConfigServer } from './ssr/remove-ts-config-server';
+import { withFallbackDocsForMigrated_6_8_To_2211_19 as withFallbackDocs } from './fallback-advice-to-follow-docs';
+import { updateAppModuleForSsr } from './ssr/update-app-module-for-ssr';
+
+/**
+ * Modernizes an application to use new Angular v17 standards.
+ */
+export function migrate(): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    return chain([
+      withFallbackDocs(updateAngularJsonForApplicationBuilder()),
+      withFallbackDocs(updateTsConfig()),
+      withFallbackDocs(updateAppModule()),
+
+      ...(isUsingSsr(tree, context)
```

Comment:
`medium:` let's consider using `checkIfSSRIsUsed` instead of providing new util
https://github.com/SAP/spartacus/blob/8f51be7ec336d0b25cce93677978cfeff4afaa51/projects/schematics/src/shared/utils/package-utils.ts#L151

Reply:
done. reused. renamed this old util to `isUsingLegacyServerBuilder`